exporters/committime/__init__.py

```diff
@@ -124,12 +126,55 @@ def git_server(self):
 
         return url
 
+    @property
+    def azure_project(self):
+        return self.__azure_project
+
     def __parse_repourl(self):
         """Parse the repo_url into individual pieces"""
         logging.debug("repo url = %s", self.__repo_url)
         if self.__repo_url is None:
             return
-        parsed = giturlparse.parse(self.__repo_url)
+        # http://user@dev.azure.com:8080/organization/project/_git/repository/
+        regex = re.compile(
```

Comment:
Is there a reason we're using a regex instead of a URL parser?

Also, are we still using `giturlparse` anywhere else? If not, can we drop the dependency?

Reply:
Just because I did not found a lib with the implementation

Yeah, we still use `giturlparse` for non azure cases